zbus_macros/src/lib.rs

```diff
@@ -289,6 +289,13 @@ pub fn proxy(attr: TokenStream, item: TokenStream) -> TokenStream {
 /// using this since it will force all interested peers to fetch the new value and hence result in
 /// excess traffic on the bus.
 ///
+/// The getter property arguments support the following `zbus` attributes:
+///
+/// * `header` - This marks the property argument to receive an optional message header associated
+///   with the D-Bus property get operation being handled. It will be `None` when the getter is
+///   being called as part of the initial object setup (before it gets registered on the bus), or
+///   when we send out property changed notifications.
```

Comment:
Good job but I think this doesn't need a special section. We should just add to the `header` arg docs below that in case of property methods, the header is optional and when its the case.